mlflow_faculty/trackingstore.py

```diff
@@ -358,7 +358,9 @@ def search_runs(self, experiment_ids, search_expressions, run_view_type):
             )
             while True:
                 list_runs_response = self._client.list_runs(
-                    self._project_id, experiment_ids=experiment_ids
+                    self._project_id,
+                    experiment_ids=experiment_ids,
+                    lifecycle_stage=run_view_type,
```

Comment:
Need to map MLflow `ViewType` to faculty `LifecycleStage`.

Reply:
We already have this mapping, so I think we can use it.